env/production/sns/terragrunt.hcl

```diff
@@ -0,0 +1,23 @@
+terraform {
+  source = "git::https://github.com/cds-snc/forms-terraform//aws/sns?ref=${get_env("TARGET_VERSION")}"
+}
+
+dependency "kms" {
```

Comment:
Same comment around `dependencies` - you just need to add a block like so above:
```hcl
dependencies {
  paths = ["../kms"]
}
```